app/human-detector-app/screens/NotifScreen.tsx

```diff
@@ -31,11 +34,20 @@ export default function NotifScreen({ navigation, route }: Props): React.ReactEl
               }}
             >
               <Text style={styles.dateText}>
-                {notification.timestamp.toLocaleDateString(undefined)}
+                {today.toLocaleDateString(undefined) ===
+                notification.timestamp.toLocaleDateString(undefined)
+                  ? 'Today'
```

Comment:
On the topic of the date debate from Discord:
```suggestion
                {today.getDay() === notification.getDay() ? 'Today'
```

Reply:
What's the range of the day? Is it day in a month, days since epoch?

Reply:
frick oops lol